### PR TITLE
Feature/brmp 3105

### PR DESCRIPTION
**Pull request by alexdntowe**
Here is the pull request for the home page

### CODE REVIEW

File: src/app/features/home/home.component.html [hunk 1]
```diff
@@ -1,77 +1,50 @@
...
           </div>
         </div>
-
-        <div class="card col-5-sm mt-2">
-          <div class="card-body">
-            This is some text within a card body.
-            <br>
-            <p class="mx-auto dx-read-more-btn pt-2"><a class="dx-button" href="#" role="button">Read More</a></p>
-
+        <div class="img-right">
```
Comment (alexdntowe) [inline src/app/features/home/home.component.html:11]: is it a custom class? if so we need to add dx-

File: src/app/features/home/home.component.html [hunk 2]
```diff
@@ -1,77 +1,50 @@
...
+          <img src="/assets/images/monuments.png" alt="washington DC monuments vector clip art background image" role="presentation">
+        </div>
+        <!--Search container-->
+        <div class="row mx-md-5">
+          <div class="col-lg-10 input-group mb-3"> 
+              <button class="dropdown btn btn-primary dropdown-toggle home-search-dropdown" type="button" data-toggle="dropdown" aria-expanded="false" aria-controls="filtersMenu"><span class="sr-only">Filters for the search, default</span>Everything
```
Comment (alexdntowe) [inline src/app/features/home/home.component.html:17]: if home-search-dropdown is a custom class, add dx-

File: src/app/features/home/home.component.html [hunk 3]
```diff
@@ -1,77 +1,50 @@
...
+              <span class="caret"></span></button>
+              <div  id="filtersMenu"  role="region" aria-label="Filter options">
+                <ul class="dropdown-menu">   
+                  <li class="nav-item home-search-filter"
```
Comment (alexdntowe) [inline src/app/features/home/home.component.html:21]: update to dx-home-search-filter

File: src/app/features/home/home.component.html [hunk 4]
```diff
@@ -1,77 +1,50 @@
...
+        <!--Search container-->
+        <div class="row mx-md-5">
+          <div class="col-lg-10 input-group mb-3"> 
+              <button class="dropdown btn btn-primary dropdown-toggle home-search-dropdown" type="button" data-toggle="dropdown" aria-expanded="false" aria-controls="filtersMenu"><span class="sr-only">Filters for the search, default</span>Everything
+              <span class="caret"></span></button>
+              <div  id="filtersMenu"  role="region" aria-label="Filter options">
+                <ul class="dropdown-menu">   
+                  <li class="nav-item home-search-filter" 
+                    *ngFor="let searchFilter of homeContent.myfilters">
+                    <a class="search-filter-a" href="searchFilter.path">{{searchFilter.filter}}</a>
```
Comment (alexdntowe) [inline src/app/features/home/home.component.html:23]: add dx-   we also have a global class for links

File: src/app/features/home/home.component.html [hunk 5]
```diff
@@ -1,77 +1,50 @@
...
+                </li>
+                </ul>
+              </div>
+            <input type="text" class="form-control home-search-input" placeholder="I'm looking for..." aria-label="Input the search criteria" aria-describedby="basic-addon2">
+            <button class="btn btn-primary rounded-0 input-group-append home-search-btn-block" type="button" aria-label="Start search">Search</button>
```
Comment (alexdntowe) [inline src/app/features/home/home.component.html:28]: add dx- to custom classes

File: src/app/features/home/home.component.html [hunk 6]
```diff
@@ -1,77 +1,50 @@
...
         </div>
-      </div>
-        
-      <div class="card col-lg m-2">
-        <div class="card-body">
-          This is some text within a card body.
-          <br>
-          <p class="mx-auto dx-read-more-btn pt-2"><a class="btn btn-primary btn-lg" href="#" role="button">Read More</a></p>
+        <div class="col-md-4 dx-vendor-cta">
+          <h2>{{homeContent.DX_home_body_header_seller}}</h2>
```
Comment (alexdntowe) [inline src/app/features/home/home.component.html:44]: use Camel case  for property names

File: src/app/features/home/home.component.ts [hunk 7]
```diff
@@ -1,15 +1,27 @@
...
   selector: 'app-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.scss']
 })
 export class HomeComponent implements OnInit {
+  
+  homeContent: any;
 
-  constructor() { }
+  private _url:string = "../assets/data/home_content.json"
```
Comment (alexdntowe) [inline src/app/features/home/home.component.ts:13]: ; is missing

File: src/app/features/home/home.component.ts [hunk 8]
```diff
@@ -1,15 +1,27 @@
...
+  
+  constructor(private http: HttpClient) {}
+
+  getHomeContent() {
+    return this.http.get(this._url);
+  }
 
   ngOnInit(): void {
+    this.getHomeContent().subscribe(resp => {
```
Comment (alexdntowe) [inline src/app/features/home/home.component.ts:22]: There is a console error on the home page.
it seems like there is a delay when the template is trying to load getHomeContent while  it is still getting the data from the http request.
You can adjust the template to only be displayed until the data is available, later we can add a spinner there.

File: src/styles.scss [hunk 9]
```diff
@@ -121,22 +121,67 @@ body {
...
+    border-color: $dx-color-primary;
+    box-shadow: none;
+	}
+}
+.btn-primary.focus {
+	color: #fff;
+	background-color: $dx-color-primary;
+  border-color: $dx-color-primary;
+  box-shadow: none;
+}
```
Comment (sperro) [inline src/styles.scss:170]: this should be appended with the custom dx class

File: src/app/features/home/home.component.html [hunk 10]
```diff
@@ -1,77 +1,50 @@
...
-            <br>
-            <p class="mx-auto dx-read-more-btn pt-2"><button class="dx-button" href="#" role="button">Read More</button></p>
-
+  <section role="search">
+    <!-- Main jumbotron for a primary search action -->
+    <div class="jumbotron jumbotron-fluid dx-home-banner-img mb-0">
+      <div class="container-fluid">
+        <div class="row mx-md-5">
+          <div class="col-lg-10">
+            <h1 class="dx-banner-title">{{homeContent.DX_home_banner_header}}</h1>
```
[the remark this answers is not in the document]
Comment (sperro) [reply src/app/features/home/home.component.html:7]: addressed

[the remark this answers is not in the document]
Comment (sperro) [reply src/app/features/home/home.component.html:7]: not in the branch. will addressed after code merged in.

File: src/app/features/home/home.component.html [hunk 11]
```diff
@@ -1,77 +1,50 @@
...
+            <p class="dx-banner-paragraph">The Federal Acquisition Service is here to <strong>help</strong>.</p>
```
[the remark this answers is not in the document]
Comment (sperro) [reply src/app/features/home/home.component.html:8]: not in the branch. will address after code is merged in

File: src/app/features/home/home.component.html (hunk 1 above)
Comment (sperro) [reply src/app/features/home/home.component.html:11]: addressed

File: src/app/features/home/home.component.html (hunk 3 above)
Comment (sperro) [reply src/app/features/home/home.component.html:21]: addressed

File: src/app/features/home/home.component.html (hunk 4 above)
Comment (sperro) [reply src/app/features/home/home.component.html:23]: addressed

File: src/app/features/home/home.component.html (hunk 6 above)
Comment (sperro) [reply src/app/features/home/home.component.html:44]: addressed

File: src/app/features/home/home.component.ts (hunk 7 above)
Comment (sperro) [reply src/app/features/home/home.component.ts:13]: addressed

File: src/app/features/home/home.component.ts (hunk 8 above)
Comment (sperro) [reply src/app/features/home/home.component.ts:22]: addressed

File: src/app/features/home/home.component.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (sperro) [reply src/app/features/home/home.component.html:17]: addressed

[the remark this answers is not in the document]
Comment (sperro) [reply src/app/features/home/home.component.html:17]: addressed

File: src/app/features/home/home.component.scss [hunk 12]
```diff
@@ -4,7 +4,7 @@
   max-width: 38rem;
 }
 
-.image-cropper {
+/*.image-cropper {
```
[the remark this answers is not in the document]
Comment (sperro) [reply src/app/features/home/home.component.scss:7]: addressed

File: src/app/features/home/home.component.scss [hunk 13]
```diff
@@ -20,4 +20,250 @@
...
+  color:$dx-color-primary-darker;
+  font-weight: 700;
+  overflow-wrap: break-word; 
+}
+.dx-banner-paragraph {
+  color:$dx-color-primary-darker;
+  font-size: 1.3rem;
+}
+.dx-home-banner-img {
+  /*background: url("/assets/images/dx_banner_img.jpg") no-repeat center center;*/
```
[the remark this answers is not in the document]
Comment (sperro) [reply src/app/features/home/home.component.scss:36]: addressed

File: src/styles.scss (hunk 9 above)
Comment (sperro) [reply src/styles.scss:170]: addressed

**Pull request merged**